planty/chats/utils.py

```diff
@@ -0,0 +1,22 @@
+
+from .models import Message, Chat
+from django.db.models import Q
+
+
+def return_list_last_messages_in_chat(request):
+    all_users_chats = Chat.objects.filter(Q(buyer=request.user) | Q(
+        post__user=request.user))
+
+    list_last_messages_in_chat = []
+    for chat in all_users_chats:
+        last_message = Message.objects.filter(
+            chat=chat).order_by('-date')[0]
+
+        if len(last_message.message_reply) > 18:
+            last_message.short = last_message.message_reply[:15] + '...'
+        else:
+            last_message.short = last_message.message_reply
+
+        list_last_messages_in_chat.append(last_message)
+    list_last_messages_in_chat[::-1]
```

Comment:
Is this reversing the list? Why do we need this? Can we add a `order_by` to the `Chat.objects...` above instead? Also, I don't think `list_last_messages_in_chat[::-1]` is reversing the list in-place so you are probably just throwing away the result of that line